docs/notebooks/batch_optimization.pct.py

```diff
@@ -204,6 +226,17 @@ def build_model(data):
     num_steps, initial_data, model, acquisition_rule=local_penalization_rule
 )
 
+# %% [markdown]
+# then with `FantasizeAcquisitionFunction`...
+
+# %%
+kriging_believer_rule = EfficientGlobalOptimization(  # type: ignore
+    num_query_points=3, builder=kriging_believer_acq
+)
+kriging_believer_result = bo.optimize(
+    num_steps, initial_data, model, acquisition_rule=kriging_believer_rule
```

Comment:
looks good (and since you're using the same num_steps you don't have to do anything to get the quickrun notebook behaviour in github)

Reply:
@uri-granta is it safe to re-use the model object that was already used in a BO loop?

Reply:
we fit the initial model at the start of each new bo optimization loop so it should be fine, right?